pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -61,7 +77,7 @@ stdenv.mkDerivation rec {
         glxinfo
         gnugrep
         gnused
-        pciutils
+        lsof
```

Comment:
`lsof` is required now:
https://github.com/flightlessmango/MangoHud/commit/ddfcb01c30790674e7316cae14bc28c4150610a4#diff-3454b244a6938af5e3c9c0d5e12d77cbda297a34aaa70e7f0841dd6678a5e735R79